cszczepaniak/cribbly-frontend
# Finish rewriting the frontend to be more testable

# What I added (link any issues addressed)
Addresses #9 
I converted everything to use sagas and redux and added tests for it all. It's a big PR because I basically touched everything.

# How to test it
You should still be able to run the app like normal against the backend and see it work. Otherwise, you can also see that the CI is running the tests and that they're all passing.
